package.json

```diff
@@ -38,7 +38,7 @@
     "Tim Sullivan <tim@elastic.co>"
   ],
   "scripts": {
-    "test": "grunt test; grunt test:visualRegression",
+    "test": "bash -c 'grunt test $@; grunt test:visualRegression $@' --",
```

Comment:
Yeah, that's a no go.  The dev tasks need to be able to run on windows and such.